test/events/podtatohead.configure-monitoring.json

```diff
@@ -0,0 +1,16 @@
+{
+  "contenttype": "application/json",
+  "data": {
+    "project": "e2e-project",
+    "service": "podtatoserver",
+    "temporaryData": {
+      "distributor": {
+        "subscriptionID": ""
+      }
+    },
```

Comment:
Btw you only need those data if you directly talk to prometheus-service. If you're sending this event to Keptn api, please remove `temporaryData`